src/VisualStudio/Core/Def/Implementation/Workspace/WorkspaceFailureOutputPane.cs

```diff
@@ -49,26 +49,58 @@ private IVsOutputWindowPane OutputPane
                 {
                     var outputWindow = (IVsOutputWindow)_serviceProvider.GetService(typeof(SVsOutputWindow));
 
-                    // Try to get the workspace pane if it has already been registered
-                    var workspacePaneGuid = s_workspacePaneGuid;
-                    var hr = outputWindow.GetPane(ref workspacePaneGuid, out _doNotAccessDirectlyOutputPane);
+                    // Output Window panes have two states; initialized and active. The former is used to indicate that the pane
+                    // can be made active ("selected") by the user, the latter indicates that the pane is currently active.
+                    // There's no way to only initialize a pane without also making it active so we remember the last active pane
+                    // and reactivate it after we've created ourselves to avoid stealing focus away from it.
+                    var lastActivePane = GetActivePane(outputWindow);
 
-                    // If the workspace pane has not been registered before, create it
-                    if (_doNotAccessDirectlyOutputPane == null || hr != VSConstants.S_OK)
+                    _doNotAccessDirectlyOutputPane = CreateOutputPane(outputWindow);
+
+                    if (lastActivePane != Guid.Empty)
                     {
-                        if (ErrorHandler.Failed(outputWindow.CreatePane(ref workspacePaneGuid, ServicesVSResources.IntelliSense, fInitVisible: 1, fClearWithSolution: 1)) ||
-                            ErrorHandler.Failed(outputWindow.GetPane(ref workspacePaneGuid, out _doNotAccessDirectlyOutputPane)))
-                        {
-                            return null;
-                        }
-
-                        // Must activate the workspace pane for it to show up in the output window
-                        _doNotAccessDirectlyOutputPane.Activate();
+                        ActivatePane(outputWindow, lastActivePane);
                     }
                 }
 
                 return _doNotAccessDirectlyOutputPane;
             }
         }
+
+        private IVsOutputWindowPane CreateOutputPane(IVsOutputWindow outputWindow)
+        {
+            // Try to get the workspace pane if it has already been registered
```

Comment:
❕ Should assert UI thread affinity (`AssertIsForeground()`)

Reply:
Fixed.